aws/v11.4.2/release.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v11.4.2
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.2.3
+  - componentVersion: 0.9.0
+    name: cert-manager
+    version: 1.0.8
+  - name: chart-operator
+    version: 0.12.4
+  - componentVersion: 1.16.5
+    name: cluster-autoscaler
+    version: 1.16.0
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.1.10
+  - componentVersion: 0.7.2
+    name: external-dns
+    version: 1.2.2
+  - componentVersion: 3.5.0
+    name: kiam
+    version: 1.3.0
+  - componentVersion: 1.9.5
+    name: kube-state-metrics
+    version: 1.1.0
+  - componentVersion: 0.3.3
+    name: metrics-server
+    version: 1.1.0
+  - name: net-exporter
+    version: 1.9.0
+  - componentVersion: 0.18.1
+    name: node-exporter
+    version: 1.2.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-cni
+    version: 1.6.0
+  - name: aws-operator
+    version: 8.7.2-anvdd
+  - name: calico
+    version: 3.10.4
+  - name: cert-operator
+    version: 0.1.0
+  - name: cluster-operator
+    version: 2.3.1-anvdd
+  - name: containerlinux
+    version: 2512.2.1
+  - name: etcd
+    version: 3.4.9
+  - name: kubernetes
+    version: 1.16.9
```

Comment:
If possible, it would also be great to have 1.16.11+ to mitigate [this CVE](https://github.com/giantswarm/giantswarm/issues/12074)

Reply:
Sure, we can include that.